.github/workflows/test-deployment.yml

```diff
@@ -22,22 +23,38 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          ref: ${{ github.ref }} 
+          ref: ${{ github.ref }}
 
-      - name: Set up NodeJS
-        uses: actions/setup-node@v3
+      - uses: actions/setup-node@v3
+        name: Set up Node.js
         with:
-          node-version: 18
+          node-version: ${{ env.NODE_VERSION }}
+          cache: npm
+          cache-dependency-path: |
+            DfE.FindInformationAcademiesTrusts/package-lock.json
+            tests/playwright/package-lock.json
 
       - name: Install dependencies
         run: npm ci
 
+      - name: Get installed Playwright version
+        id: playwright-version
+        run: echo "PLAYWRIGHT_VERSION=$(node -e "console.log(require('./package-lock.json').packages['node_modules/@playwright/test'].version)")" >> $GITHUB_ENV
+
+      - name: Cache playwright binaries
+        uses: actions/cache@v3
+        id: playwright-cache
+        with:
+          path: |
+            ~/.cache/ms-playwright
+          key: ${{ runner.os }}-playwright-${{ env.PLAYWRIGHT_VERSION }}
```

Comment:
I don't think this will work.... the browsers downloaded (aka the playwright binaries) correlate to the current version of the browsers released (e.g. Chrome 115.0.5790.110) not the version of playwright in the project.

Reply:
I think it might be easier for us to use a `Y-m-d` date stamp as a cache key so the cache invalidates after 24H. It's not ideal, but it should mean sequential pushes in the same day use the same browsers. Would that work for you?